chaco/data_view.py

```diff
@@ -208,10 +208,65 @@ class DataView(OverlayPlotContainer):
     bgcolor = "white"
 
     #: Padding defaults.
-    padding_top = 50
-    padding_bottom = 50
-    padding_left = 50
-    padding_right = 50
+    padding_top = Property(
```

Comment:
if explicitly set (or if `padding` trait is sett, which then explicitly sets these), that value will always be used.  Setting sets the shadow traits (with leading underscore), and in the getters we explicitly check if the shadow trait is set and return it.  Otherwise we then check for axis titles and adjust as needed either returning 80 or 50 as a default.

Note I am just now considering if a user explicitly sets one off these traits to 0 I dont think it will work as then we will not enter the `if getattr(self, MAP[side])` block.  This is a bug that needs fixing.